Resources/Prototypes/Recipes/Lathes/security.yml

```
@@ -1446,3 +1446,65 @@
+    Steel: 30000
+    Plastic: 20000
+
+- type: latheRecipe
+  id: SmartgunMagazineLightRifleBoxShrapnel
+  result: SmartgunMagazineLightRifleBoxShrapnel
+  category: Ammo
+  completetime: 10
+  materials:
+    Steel: 35000
+    Plastic: 35000
+    Plasteel: 35000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)